generative/maisi/data/README.md

```diff
@@ -0,0 +1,18 @@
+# Medical AI for Synthetic Imaging (MAISI) Data Preparation
+
+Disclaimer: We are not the host of the data. Please make sure to read the requirements and usage policies of the data and give credit to the authors of the dataset!
+
+### 1 VAE training Data
+The VAE training dataset used in MAISI contains 37243 CT traiing data, 17887 CT validation data
+
+### 2 Unconditional Diffusion model training Data
```

Comment:
consider removing "Unconditional". The unet is conditioning on spacing and body region.

Reply:
Done. Thanks